docs/application/dotnet/guides/user-interface/nui/relative-layout.md

```diff
@@ -21,16 +21,32 @@ The following figure shows the relative layout position of rectangles with `Targ
         <RelativeLayout/>
     </View.Layout>
 
-    <View x:Name="RedRect"   BackgroundColor="1,0,0,1" WidthSpecification="100" HeightSpecification="100"/>
-    <View x:Name="GreenRect"  BackgroundColor="0,1,0,1" HeightSpecification="100"
-          RelativeLayout.LeftTarget ={x:Reference Name=RedRect}" RelativeLayout.RightTarget="{x:Reference Name=BlueRect}"
-          RelativeLayout.FillHorizontal="true" RelativeLayout.LeftRelativeOffset="1.0"/>
-    <View x:Name="BlueRect"  BackgroundColor="0,0,1,1" WidthSpecification="100" HeightSpecification="100"
-          RelativeLayout.RightRelativeOffset="1.0" RelativeLayout.LeftRelativeOffset="1.0"
-          RelativeLayout.HorizontalAlignment="End"/>
-    <View x:Name="YellowRect"  BackgroundColor="1,1,0,1" HeightSpecification="100"
-          RelativeLayout.LeftTarget ="{x:Reference Name=Parent}"
-          RelativeLayout.RightTarget="{x:Reference Name=BlueRect}"      RelativeLayout.FillHorizontal="true"/>
+    <View x:Name="RedRect"
+        BackgroundColor="1,0,0,1"
+        WidthSpecification="100"
+        HeightSpecification="100"/>
+
+    <View x:Name="GreenRect"
+        BackgroundColor="0,1,0,1" HeightSpecification="100"
+        RelativeLayout.LeftTarget={x:Reference Name=RedRect}"
+        RelativeLayout.RightTarget="{x:Reference Name=BlueRect}"
+        RelativeLayout.FillHorizontal="true"
+        RelativeLayout.LeftRelativeOffset="1.0"/>
+
+    <View x:Name="BlueRect"
+        BackgroundColor="0,0,1,1"
+        WidthSpecification="100"
+        HeightSpecification="100"
+        RelativeLayout.RightRelativeOffset="1.0"
+        RelativeLayout.LeftRelativeOffset="1.0"
+        RelativeLayout.HorizontalAlignment="End"/>
+
+    <View x:Name="YellowRect"
+        BackgroundColor="1,1,0,1"
+        HeightSpecification="100"
+        RelativeLayout.LeftTarget="{x:Reference Name=Parent}"
+        RelativeLayout.RightTarget="{x:Reference Name=BlueRect}"
+        RelativeLayout.FillHorizontal="true"/>
 </View>
```

Comment:
Could you clarify if this comment is fixed, I am unable to figure out?

[https://github.com/Samsung/tizen-docs/pull/1521#discussion_r742537933](https://github.com/Samsung/tizen-docs/pull/1521#discussion_r742537933)